sdk/storage/azure-storage-perf/src/main/java/com/azure/storage/blob/perf/BlobPerfStressOptions.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.perf;
+
+import com.azure.storage.StoragePerfStressOptions;
+import com.beust.jcommander.Parameter;
+
+public class BlobPerfStressOptions extends StoragePerfStressOptions {
+
+    @Parameter(names = { "--encryption-version" })
+    private String encryptionVersion;
```

Comment:
this can be enum, JCommander should be able to parse it.

See https://github.com/Azure/azure-sdk-for-java/blob/741fa238b645bf721e08c8d1f7dd2319991e7329/sdk/core/azure-core-perf/src/main/java/com/azure/core/perf/core/CorePerfStressOptions.java#L17
for reference.

Make sure to default to null.